doc/RecordingControl.xml

```diff
@@ -2039,13 +2040,119 @@ secfrac = "." 1*6DIGIT</programlisting>
         </variablelist>
       </para>
       <para>
-        The device shall create an individual initialization vector for each segment.
-        The device shall encrypt with the <literal>Key</literal> configured for the encryption entry as the encryption key.
+        The device shall create a unique initialization vector for each segment.
         The device shall interpret the <literal>KID</literal> configured for the encryption entry as a 16-byte hexadecimal value and use this value as the key identifier.
```

Comment:
This line seems a bit redundant given that KID is now always a 16-byte hexadecimal value. Also, I think this paragraph should mention the use of `CertificateID` and when to use it. Or maybe we want to move everything into the "Standard CENC system" / "Key rotation system" sections below and only state which system to use in this paragraph?